### DISCUSSION TITLE
is it possible to add stadia pro compatibility?

### DISCUSSION THREAD

**Discussion by emulation** [ndeadly/MissionControl#513] [General]
I have a stadia pro controller (founders edition)  but in my country peru is not available the service so i can't activate the service but i have the controller and i want to use it.... with my nintendo switch as a pro controller.
please if is possible please add compatibility.

Thanks

**Comment by impeeza** [comment]
Stadia have Bluetooth conectivity?  Datasheet only show WiFi or USB connections

**Comment by impeeza** [comment]
My bad, indeed it have Bluetooth radio but the firmware only have BLE implemented:

https://support.google.com/stadia/answer/9338851?hl=en

`Product contains Bluetooth Classic radio. No Bluetooth Classic functionality is enabled at this time. Bluetooth Classic functionality may be implemented at a later date. `


So, by now no support will be possible

**Reply by ndeadly** [reply]
It's worse than that. Not only does it use BLE (and therefore currently can't be supported) but it doesn't even support HID over Bluetooth. Supposedly the Bluetooth connection is only used for some kind of initial setup/synchronisation of the controller. It's been said that it's possible that they could enable HID in a firmware update, but to date I don't think anything has been done about that.

**Comment by ndeadly** [comment]
Unfortunately it's not possible to support the controller via Bluetooth currently (see my response to Impeeza). When mission control supports USB I will be able to support it in wired mode though I think.

**Comment by ndeadly** [comment]
Good news, Google is releasing Bluetooth support for the Stadia controller soon!. Let's hope it's not using Bluetooth LE

https://community.stadia.com/t5/Stadia-General/A-Gift-from-the-Stadia-Team-amp-Bluetooth-Controller/m-p/85936

**Reply by swierszczyk18** [reply]
Any news what Bluetooth does it use and will mission control support it? Would be great to use those stadia controllers on my switch.

**Reply by saintyuk** [reply]
Probably best to follow: https://github.com/ndeadly/MissionControl/issues/568

**Reply by swierszczyk18** [reply]
TY, very helpful:)

**Comment by pixel2user** [comment]
Just a info regarding USB support: It's possible to use the Google Stadia Controller on a modded Switch with this config.ini https://github.com/o0Zz/sys-con/issues/82#issuecomment-3531272972
